src/testdir/test_cmdline.vim

```diff
@@ -4011,4 +4011,36 @@ func Test_cd_bslash_completion_windows()
   let &shellslash = save_shellslash
 endfunc
 
+func Test_msghistory()
+  " NOTE: 'Messages maintainer:' message must be filtered
+  function! GetMessages() abort
```

Comment:
```suggestion
  function GetMessages() abort
```

No use `!`, because it would cause an error if already exists.

Reply:
Isn't it the other way around?

Reply:
Temporary functions should be cleaned up, if another test forgets to clean up or if there are global helper functions we can catch the overloading error.

Reply:
Yes